pkg/skaffold/kubernetes/log.go

```diff
@@ -156,14 +151,31 @@ func (a *LogAggregator) streamContainerLogs(ctx context.Context, pod *v1.Pod, co
 	tr, tw := io.Pipe()
 	cmd := exec.CommandContext(ctx, "kubectl", "logs", sinceSeconds, "-f", pod.Name, "-c", container.Name, "--namespace", pod.Namespace)
 	cmd.Stdout = tw
-	go util.RunCmd(cmd)
+	go func() {
+		util.RunCmd(cmd)
+		tw.Close()
+	}()
 
-	color := a.colorPicker.Pick(pod)
+	podColor := a.colorPicker.Pick(pod)
 	prefix := prefix(pod, container)
 	go func() {
-		if err := a.streamRequest(ctx, color, prefix, tr); err != nil {
+		if err := a.streamRequest(ctx, podColor, prefix, tr); err != nil {
 			logrus.Errorf("streaming request %s", err)
 		}
+
+		// TEMP: Get the updated state of the container
+		client, _ := Client()
+		pod, _ := client.CoreV1().Pods(pod.Namespace).Get(pod.Name, meta_v1.GetOptions{})
```

Comment:
i think we shd check here is pod is not terminated. 
If pod is killed, we should emit the output for `kubectl describe pod`